src/app/modules/showcase-ui/examples/table-example/table-example.component.scss

```diff
@@ -0,0 +1,10 @@
+::ng-deep lab900-table {
+  tr.lab900-table__pre-footer-title-row {
+    background-color: #b3e6e4;
+    text-align: center;
+
+    td.lab900-table__pre-footer-title-cell {
+      text-align: center;
+    }
+  }
+}
```

Comment:
I would left align it, it looks a bit out of place.
I would also make it background color?
@lab900-senn-lambrecht what do you think

Reply:
This is just example styling, doesn't matter that much I guess.

Reply:
yes it was just to show what you can do